charts/nutanix-capx/templates/k0s-controlplane-template.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.modes.standalone.enabled }}
+---
+apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+kind: K0sControlPlaneTemplate
+metadata:
+  name: {{ include "nutanix-capx.standalone.controlPlaneTemplateName" . }}
+  labels:
+    {{- include "nutanix-capx.labels" . | nindent 4 }}
+spec:
+  template:
+    spec:
+      version: {{ .Values.k0s.version | quote }}
+      k0sConfigSpec:
+        args:
+          - --enable-worker
+          - --enable-cloud-provider
+          - --kubelet-extra-args=--cloud-provider=external
+          - --disable-components=konnectivity-server
+          - --no-taints
+          {{- range .Values.k0s.args }}
+          - {{ . | quote }}
+          {{- end }}
+        k0s:
+          apiVersion: k0s.k0sproject.io/v1beta1
+          kind: ClusterConfig
+          metadata:
+            name: k0s
+          spec:
+            telemetry:
+              enabled: false
+            api:
+              extraArgs:
+                anonymous-auth: "true"
+                kubelet-preferred-address-types: "ExternalIP,Hostname,InternalDNS,ExternalDNS"
+            network:
+              provider: calico
+              calico:
+                mode: vxlan
```

Comment:
Calico network config is different in-between templates, earlier template had more fields, here we don't have any, I suggest make all fields configurable as free-type from Helm Values or mirror same fields here so we would have uniform Calico configuration options across charts